codegen/module_system.go

```diff
@@ -34,7 +34,11 @@ import (
 
 // NewDefaultModuleSystem creates a fresh instance of the default zanzibar
 // module system (clients, endpoints)
-func NewDefaultModuleSystem(h *PackageHelper) (*module.System, error) {
+func NewDefaultModuleSystem(
+	configDirName string,
+	middlewareConfig string,
+	h *PackageHelper,
```

Comment:
Honestly, I didn't know a good way to get a hold of the middleware config, I know it feels awkward to pass in as arguments, it doesn't feel belong to PackageHelper either.